integral_view/templates/view_zfs_dataset.html

```diff
@@ -11,7 +11,10 @@
     <div class="btn-group btn-group-sm pull-right"  >
       <a class="btn btn-default dropdown-toggle" data-toggle="dropdown" href="#"> <i class="fa fa-cog fa-fw"></i>&nbsp;Actions &nbsp;<span class="fa fa-caret-down" title="Toggle dropdown menu"></span> </a>
       <ul class="dropdown-menu">
-        <li><a href="/update_zfs_dataset?name={{name}}&pool_name={{pool}}" > <i class="fa fa-cog fa-fw"></i>&nbsp;Change {%if properties.type.value == 'filesystem'%} dataset {%elif properties.type.value == 'volume'%} block device volume {%endif%} properties</a> </li>
+        {%if request.user.username == 'superadmin'%} 
+          <li><a href="/update_zfs_dataset?name={{name}}&pool_name={{pool}}" > <i class="fa fa-cog fa-fw"></i>&nbsp;Update {%if properties.type.value == 'filesystem'%} dataset {%elif properties.type.value == 'volume'%} block device volume {%endif%} basic properties</a> </li>
```

Comment:
I was wondering about it. The update_zfs_dataset actually changes the dedup, compress and the quota params which should not be changed flippantly so thought I would encapsulate that as well.